.github/workflows/check-style.yml

```diff
@@ -0,0 +1,13 @@
+name: test-clang-format
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+    - uses: yshui/git-clang-format-lint@master
```

Comment:
This action also commit after formatting?

Reply:
@Mihaylov93 rename build to test-clang-format

```yaml
jobs:
  test-clang-format:
```